src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall(){
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
```

Comment:
Ensure that the `ColorSupplier` class is implemented correctly and that the `getRandomColor()` method returns a random color from the `Color` enum. This is crucial for the `getRandomBall()` method to function as expected.